### PR TITLE
Feautre/readme

### PR DESCRIPTION
**Pull request by forgondolin**
Hello, I'm justing posting the PR for the README, tell if it is good

### CODE REVIEW

**Comment by pschoffer** [comment]
Hi @forgondolin ,

this is great thank you! 

- [ ] However, it would be great if you could replace the instructions section with using `gitpress` instead of manual install. That is Something like `Example Usage` in here: https://github.com/pschoffer/gitpress

- [ ] Also, please remove the old README.html that you are replacing.

**Comment by forgondolin** [comment]
@pschoffer . Ok, I'll do that, thanks for the review

**Comment by forgondolin** [comment]
I've added the initialization with docker, but i've left the manual install too. What do you think?

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+       - 80:80
+     environment:
+       GITPRESS_MODE: FETCH
+       WORDPRESS_URL: http://localhost/
+       GIT_ORIGIN: github.com/pschoffer/gitpress-example.git
+ ```
+
+The admin login credentials are `gitpress:pass`		
+
+## Updating
```
Comment (pschoffer) [inline readme.md:47]: Please drop all the sections bellow. Those seems to be related to wordpress itself, but this is only example repo meant to be used as demo for the gitpress itself. 

Although there is some value in comming up with migration flow, it will need to be more complex than that (leveraging the SEED mode of gitpress and then applying changes and persisting them to git).

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# GitPress Example
+
+The objective here is to create example project for the ![Gitpress platform](https://github.com/pschoffer/gitpress).
+Its very easy to install and you can try it easily home.
+
+## Instalation
+
+ ### Manual Instalation
```
Comment (pschoffer) [inline readme.md:8]: Please drop the manual instalation. Have you tried if that approach actually works?

The thing is that GitPress has "integrated" database. It is not meant to connect to any other database, these steps are addressing configuring wordpress with database and creating DB schema which is something this project is aiming to avoid. (by doing it in kinda integrated transparent way)

**Review by pschoffer — CHANGES_REQUESTED** [review]
Hi @forgondolin ,
again thanks a lot for your work. I left some comments in the file. I am sorry, that I might have not been super clear in the beginning.

It seems the text you wrote is kinda related to the Wordpress only, but it might work quite differently here as there is no database from the users perspective. Let's keep the readme simple for now. 

Please dont hesitate to fire some question.

File: readme.md (hunk 1 above)
Comment (forgondolin) [reply readme.md:47]: Just tu be clear. 
Drop the whole Updating section and also the "...The admin login credentials..." line?

File: readme.md (hunk 2 above)
Comment (forgondolin) [reply readme.md:8]: I got it. 
Its a very nice project, simplifies a lot the Wordpress experience. It reminds me of Fission codes platform on IPFS that let you build you front page without bothering about the backend and the publishing.

File: readme.md (hunk 1 above)
Comment (pschoffer) [reply readme.md:47]: > Just tu be clear.
Drop the whole Updating section and also the "...The admin login credentials..." line?

* Please keep the The admin login credentials are `gitpress:pass`, as that is related to the docker install.
* Drop the update
* Drop Migrating from other systems
* Drop System Requirements (or change to just docker I guess)
* Drop Recommendations
* Drop Online Resources and Final Notes
* Drop Share the Love (or rephrase it to apply to Gitpress instead of WordPress, as the same is true for this project as well:))
* In Licence change Wordpress to either "This Gitpress example" or something along those lines (or drop)

File: readme.md (hunk 2 above)
Comment (pschoffer) [reply readme.md:8]: Great:) Yeah to be honest I don't know much about WordPress and I knew much less when I started this. But on the other hand, this is just mashing together that awesome VersionPress plugin, WordPress and some docker goodies to standalone thing:)

You know, the regular "standing on the shoulders of giants" approach. It is awesome that it got you interested in contributing \o/.

**Comment by forgondolin** [comment]
This is my first time addressing to Hackthober fest repos, so I actually dont know too much about it. Glad to help. I'll share the GitPress project to more people

**Comment by pschoffer** [comment]
This is great. I am happy to merge it.

However, I don't know if it counts towards hacktoberfest if the PR was open before October though, can you see it counted there already? Otherwise we could decline this PR and you could open a new one in few days if you want to (same for your other PRs).

**Comment by forgondolin** [comment]
Yeah, none count, but no problems, I can reopen all of them in a couple of days, if it isnt a problem to you, of course

**Comment by pschoffer** [comment]
Sure sounds good, then feel free to close these and reopen then. I guess that `Close with comment` button will do that.
